dataflow/src/main/java/org/checkerframework/dataflow/expression/JavaExpression.java

```diff
@@ -914,4 +916,85 @@ private static boolean isVarArgsInvocation(
     return TypesUtils.getArrayDepth(ElementUtils.getType(lastParamElt))
         != TypesUtils.getArrayDepth(lastArgType);
   }
+
+  /**
+   * Find the declaration of the receiver of a method call in a method tree.
+   *
+   * <p>The receiver should appear in one of two places, either as a formal parameter to the method,
+   * or as a local variable.
```

Comment:
Can it be a field?

Reply:
No. The case in which the explicit annotation may appear on a field declaration is handled [in this code](https://github.com/typetools/checker-framework/blob/0f468a00564fd02d7f56c1ad32e95cb55f6f5f33/checker/src/main/java/org/checkerframework/checker/optional/OptionalTransfer.java#L250-L252).